dev-docs/source/ReleaseChecklist.rst

```diff
@@ -179,12 +227,12 @@ Finalise
 
 * Send an email, including the text of the release notes, to the
   following lists
-* ``mantid-announce@mantidproject.org``
 * ``nobugs@nobugsconference.org``
 * ``news@neutronsources.org``
 * ``neutron@neutronsources.org``
+* Also post the contents of the message on General on Slack
```

Comment:
I think we have been using Annoucements for this too.

Reply:
Good point I've changed this